frontend/src/components/InlineMessage/InlineMessage.tsx

```diff
@@ -0,0 +1,39 @@
+import ReactMarkdown from 'react-markdown'
+import { Box, Flex, Icon, useMultiStyleConfig } from '@chakra-ui/react'
+
+import { BxsErrorCircle, BxsInfoCircle } from '~/assets/icons'
+import { InlineMessageVariant } from '~/theme/components/InlineMessage'
+
+import { useMdComponents } from '~hooks/useMdComponents'
+
+export type InlineMessageProps = {
+  variant: InlineMessageVariant
+  children: string
+  useMarkdown: boolean
+}
+
+export const InlineMessage = ({
+  variant = 'info',
+  children,
+  useMarkdown = false,
+}: InlineMessageProps): JSX.Element => {
+  const styles = useMultiStyleConfig('InlineMessage', { variant })
+
+  const mdComponents = useMdComponents(styles)
+
+  return (
+    <Box __css={styles.messagebox}>
```

Comment:
is there a reason for this outer box?

Reply:
I think messagebox and item styles can be combined into a single `Box` component

Reply:
thanks! combined the two